src/panels/lovelace/cards/hui-iframe-card.ts

```diff
@@ -0,0 +1,82 @@
+import { html, LitElement } from "@polymer/lit-element";
+
+import "../../../components/ha-card.js";
+
+import { HassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+import { LovelaceCard, LovelaceConfig } from "../types.js";
+
+interface Config extends LovelaceConfig {
+  aspect_ratio?: string;
+  title?: string;
+  url?: string;
```

Comment:
URL should be required.

Reply:
Yes. Should an error be thrown if a required field is not given? I made the field required `url: string;` but no errors are thrown if I don't include it in my `ui-lovelace.yaml`

Reply:
Yea. You would `throw new Error("URL required")`

Reply:
That will stop it from continuing and it will not render anything